gqcp/include/Mathematical/Representation/LeviCivitaTensor.hpp

```diff
@@ -0,0 +1,71 @@
+// This file is part of GQCG-GQCP.
+//
+// Copyright (C) 2017-2020  the GQCG developers
+//
+// GQCG-GQCP is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GQCG-GQCP is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with GQCG-GQCP.  If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+
+#include "Mathematical/Representation/Tensor.hpp"
+
+
+namespace GQCP {
+
+
+/**
+ *  The antisymmetric rank-three Levi-Civita tensor.
+ */
+class LeviCivitaTensor {
+private:
+    // The values of the Levi-Civita tensor.
+    Tensor<double, 3> epsilon;
```

Comment:
Why do you use a tensor containing doubles instead of integers, which are smaller in size?
Does this have something to do with possible conversion issues later on?

Reply:
That's a good question. I guess we could take the scalar type (`double` or `int` or anything else) as a template parameter for this class, so we can accommodate both `LeviCivitaTensor<double>` and `LeviCivitaTensor<int>`.

@AaronDC60 Would you like to implement this changes in this branch?

Reply:
Sure, I will give it a try.

Reply:
You did a great job, @AaronDC60! One thing that I see in the commit history is that your GitHub account does not seem linked to your git commits, as your profile avatar does not appear in the commit timeline. Could you fix this issue?

Reply:
I configured my git on the hpc, so this issue should now be fixed.